lldb/include/lldb/API/SBInstructionList.h

```diff
@@ -62,8 +67,8 @@ class LLDB_API SBInstructionList {
   friend class SBTarget;
 
   void SetDisassembler(const lldb::DisassemblerSP &opaque_sp);
-  bool GetDescription(lldb_private::Stream &description);
-
+  bool GetDescription(lldb_private::Stream &description,
+                      lldb::SBExecutionContext *exe_ctx = nullptr);
```

Comment:
I guess this can now take the private type (i.e. `ExecutionContext*`). All the SB API objects are PIMPL objects so it feels a bit weird to take their address.

Reply:
:100: